src/components/Review/ReviewSectionRowLastAction.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Grid } from 'semantic-ui-react'
+import getSimplifiedTimeDifference from '../../utils/dateAndTime/getSimplifiedTimeDifference'
+import { ReviewSectionComponentProps } from '../../utils/types'
+import strings from '../../utils/constants'
+
+const ReviewSectionRowLastDate: React.FC<ReviewSectionComponentProps> = ({
+  action,
+  thisReview,
+  assignment,
+}) => {
+  const getContent = () => {
+    switch (action) {
+      case 'canContinue': {
+        return (
+          <p>{`${strings.REVIEW_STARTED}: ${getSimplifiedTimeDifference(
+            thisReview?.timeStatusCreated
+          )}`}</p>
+        )
+      }
+      case 'canView': {
+        return (
+          <p>{`${strings.REVIEW_SUBMITTED}: ${getSimplifiedTimeDifference(
+            thisReview?.timeStatusCreated
+          )}`}</p>
+        )
+      }
+      case 'canStartReview': {
+        return (
+          <p>{`${strings.ASSIGNED}: ${getSimplifiedTimeDifference(assignment.timeCreated)}`}</p>
+        )
+      }
```

Comment:
```suggestion
      case ReviewAction.canContinue: {
        return (
          <p>{`${strings.REVIEW_STARTED}: ${getSimplifiedTimeDifference(
            thisReview?.timeStatusCreated
          )}`}</p>
        )
      }
      case ReviewAction.canView: {
        return (
          <p>{`${strings.REVIEW_SUBMITTED}: ${getSimplifiedTimeDifference(
            thisReview?.timeStatusCreated
          )}`}</p>
        )
      }
      case ReviewAction.canStartReview: {
        return (
          <p>{`${strings.ASSIGNED}: ${getSimplifiedTimeDifference(assignment.timeCreated)}`}</p>
        )
      }
```
Also need to import `ReviewAction` from types

Reply:
9800aa9, i've made a seperate commit as it was easier to keep track of compilation error and do a fix all at once